src/wiring.c

```diff
@@ -106,10 +107,21 @@ static int wiring_parseextparms(const PROGRAMMER *pgm, const LISTID extparms) {
       pmsg_notice2("%s(): delay set to %d ms\n", __func__, val);
       WIRINGPDATA(pgm)->delay = val;
       continue;
-    } else if (str_eq(extended_param, "help")) {
+    }
+    else if(str_starts(extended_param, "no_autoreset")) {
+      if(!str_eq(extended_param, "no_autoreset")) {
+        pmsg_error("invalid -xno_autoreset value %s. Use -xno_autoreset\n", extended_param);
+        rv = -1;
+        break;
+      }
+      WIRINGPDATA(pgm)->autoreset = false;
+      continue;
+    }
+    else if (str_eq(extended_param, "help")) {
```

Comment:
I know `<arg>` below is in existing code. But it's always nicer to be as specific as possible. Arg can be anything, but `<n>` alludes to this being a number. So  `-xsnooze=<n> Wait snooze <n> ms` is defo more specific. Same with `delay` below.